diff --git a/dist/esm/node/runtime/renderPage/analyzePage.js b/dist/esm/node/runtime/renderPage/analyzePage.js
index e34c750b2c5d38ceab338393fc366efe90e417b2..48f151120ed7f1a2da08846d68e81dec5785d734 100644
--- a/dist/esm/node/runtime/renderPage/analyzePage.js
+++ b/dist/esm/node/runtime/renderPage/analyzePage.js
@@ -13,6 +13,7 @@ async function analyzePage(pageFilesAll, pageConfig, pageId, globalContext) {
             clientEntries.push(clientFilePath);
         if (isClientRuntimeLoaded)
             clientEntries.push(getVikeClientEntry(isClientRouting));
+      console.log('clientEntries', clientEntries);
         const clientDependencies = [];
         clientDependencies.push({
             id: getVirtualFileIdPageConfigValuesAll(pageConfig.pageId, true),
